src/SessionManager.php

```diff
@@ -139,10 +139,6 @@ public function start($preserveStorage = false)
 
         session_start();
 
-        if ($oldSessionData instanceof Traversable) {
```

Comment:
If we remove this part do we then need line 12 in that file? (Importing of `Traversable`)

Reply:
changed other Traversable to use import instead of FQCN